src/learn/zkcloud/README.md

```diff
@@ -1 +1,6 @@
 # zkCloud
+zkCloud is the infrastructure that allows Aleo to enable decentralized private computations. This infrastructure is made up of two parts, `snarkVM` - zero knowledge off chain execution client and `snarkOS` which is the network client that verifies that off chain computation has been done correctly by verifying the `zksnark` that has been submitted.
+
+Together `snarkVM` and `snarkOS` allow individuals (shielded identities) to remain private and use this off chain computation (shielded transactions) to interact with a given decentralized application (program) on Aleo. On the other hand it also enables developers to create these privacy preserving decentralized applications through using leo or aleo instructions and deploying the program to the network.
```

Comment:
The parenthesis explaining shielded identities and shielded transactions are a bit misleading. 
A shielded transactions is one where the owner and amount are encrypted, not an off-chain computation as written here. A transaction can be shielded (aka private) due to the Record Model and other architectural choices of Aleo, executing computation off-chain is only part of what enables a shielded transaction.

Reply:
I like how this article explains the intuition of Aleo's zkCloud without going deep into the technical explanation. We could use some of this explanations:
https://aleo.org/post/zkcloud/